src/main/java/de/hysky/skyblocker/skyblock/auction/AuctionBrowserScreen.java

```diff
@@ -352,7 +352,7 @@ private void parsePage(ItemStack stack) {
             List<Text> tooltip = ItemUtils.getNbtTooltips(stack);
             String str = tooltip.get(0).getString().trim();
             str = str.substring(1, str.length() - 1); // remove parentheses
-            String[] parts = str.split("/"); // split the string
+            String[] parts = str.split("/"); // targetSize the string
```

Comment:
Intellij refactoring moment
```suggestion
            String[] parts = str.split("/"); // split the string
```